<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationSettingsDialog</class>
 <widget class="QDialog" name="NavigationSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigation Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="container" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="home_label">
        <property name="text">
         <string>Home position</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="home_set_button">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="home_reset_button">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="presets_label">
        <property name="text">
         <string>Presets</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="presets_combobox"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>First Person / Flight</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="speed_label">
           <property name="text">
            <string>Base speed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="speed_spinbox">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="tick_label">
           <property name="text">
            <string>Speed tick</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="tick_spinbox"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="tick_multiplier_label">
           <property name="text">
            <string>x512</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="tick_startup_label">
           <property name="text">
            <string>On startup</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="tick_startup_spinbox"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="tick_startup_multiplier_label">
           <property name="text">
            <string>x1024</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Collisions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="collision_checkbox">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="collision_startup_checkbox">
           <property name="text">
            <string>On startup</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="collision_radius_label">
           <property name="text">
            <string>Radius</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="collision_radius_spinbox">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QGroupBox" name="ground_group">
        <property name="title">
         <string>Ground mode</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="ground_checkbox">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="ground_startup_checkbox">
           <property name="text">
            <string>On startup</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="height_label">
           <property name="text">
            <string>Eye height</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="height_spinbox">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="gravity_acceleration_label">
           <property name="text">
            <string>Gravity</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="gravity_acceleration_spinbox">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="gravity_speed_label">
           <property name="text">
            <string>Max fall speed</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="gravity_speed_spinbox">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>NavigationSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>NavigationSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
